examples/yara_matches.py

```diff
@@ -0,0 +1,61 @@
+"""Run a set of Yara Rule matches on Extracted Files"""
+from __future__ import print_function
+import os
+import sys
+import time
+import argparse
+from pprint import pprint
+
+# Local imports
+from brothon import bro_log_reader
+from brothon.utils import yara_rules, dir_watcher, file_utils
+
+def yara_match(file_path, rules):
+    """Callback for a newly extacted file"""
+    print('New Extracted File: {:s}'.format(file_path))
+    print('Mathes:')
+    pprint(rules.match(file_path))
+
+if __name__ == '__main__':
+    """Run a set of Yara Rule matches on Extracted Files"""
+    from shutil import copyfile
+
+    # Collect args from the command line
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-y', '--yara-dir', type=str, help='Specify the yara rules directory (e.g. /full/path/to/yara/rules)')
+    parser.add_argument('-e', '--extract-dir', type=str, help='Specify the Bro extract_files directory (e.g. /full/path/to/bro/extract_files)')
+    args, commands = parser.parse_known_args()
+
+    # Check for unknown args
+    if commands:
+        print('Unrecognized args: %s' % commands)
+        sys.exit(1)
+
+    # If no args just call help
+    if len(sys.argv) == 1:
+        parser.print_help()
+        sys.exit(1)
+
+    # Sanity check that the args exist and are directories
+    if not os.path.isdir(args.yara_dir):
```

Comment:
Looks like you will fail if it is not a dir - you would prob need to change this check?